scm-ui/ui-components/src/MarkdownView.tsx

```diff
@@ -99,6 +99,14 @@ class MarkdownView extends React.Component<Props> {
     }
   }
 
+  enrichCommitMentionsWithInternalLinks = (content: string) => {
+    const regex = new RegExp("([\\w\\d]*)\\/([\\w\\d]*)@([\\w\\d]*)\\w+");
+    return content.replace(regex, match => {
+      const splittedMatch = match.split("@");
```

Comment:
If we are already on regex, why not use groups?

Reply:
I really tried but i can't get it work. If I use group matcher the regExp won't match anymore.